tutorials/packaging/snap-a-website.md

```diff
@@ -0,0 +1,670 @@
+---
+id: snap-a-website
+summary: In this tutorial, you’ll learn how to create a desktop web app with Electron and package it as a snap.
+categories: packaging
+tags: snapcraft, snap, beginner, nodejs, electron, website
+status: Published
+feedback Link: https://github.com/ubuntu/codelabs/issues:
+author: David Callé <david.calle@canonical.com>
+difficulty: 2
+published: 2017-11-14
+
+---
+
+# Snap a website with Electron
+
+## Overview
+Duration: 1:00
+
+Turning your website into a desktop integrated app is a relatively simple thing to do, but distributing it as such and making it noticeable in app stores is another story. This tutorial will show you how to leverage Electron and snaps to create a website-based desktop app from scratch and publish it on a multi-million users app store shared between many Linux distributions.
```

Comment:
publish it on a multi-million **user** 
Also, do we want to say 'app store', I think just 'store would do fine